kompose.spec

```diff
@@ -190,11 +185,15 @@ export GOPATH=$(pwd):%{gopath}
 export GOPATH=$(pwd):$(pwd)/vendor:%{gopath}
```

Comment:
> Also, isn't the vendor directory automatically detected and used by go (maybe not in the past but definitely with 1.6+)?

Yes, `go build` 1.6+ looks for libraries in vendor dir first, if it can't find given library in vendor than it looks to $GOPATH